web/pom.xml

```diff
@@ -317,7 +317,54 @@
     <dependency>
       <groupId>org.mapfish.print</groupId>
       <artifactId>print-lib</artifactId>
-      <version>${print-lib.version}</version>
+      <!-- why no exclusions like in core/pom.xml? -->
```

Comment:
This approach has the risk of introducing conflicts (for example with `guava`,`spring-framework` and `batik`).

Reply:
@fgravin your name was given to me to ask about this?

Reply:
@fxprunayre you added the excluses in core/pom.xml can you advise please.

Reply:
From https://github.com/geonetwork/core-geonetwork/commit/e351a608cb50ece5da6220f8b40210565dfba420 it looks like it was related to "All http requests now use one configurable object allowing mocking and testing objects that need to make https request.". 
Exclusions were also updated in https://github.com/geonetwork/core-geonetwork/commit/0d462e1ce8793d8acfaa4ff5aebfaee4593d03f8
Not an expert on this and no objection to make it consistent on both modules.

Reply:
I would feel more comfortable with that approach (so we are controlling our dependencies directly). I do not want to hamper camp2camp folks working on print-lib if this is a workflow they are maintaining...